tests/manage/pv_services/test_resource_deletion_during_pvc_pod_deletion_and_io.py

```diff
@@ -434,6 +435,7 @@ def test_disruptive_during_pod_pvc_deletion_and_io(
 
         log.info("Fetching IO results from the pods.")
         for pod_obj in io_pods:
+            log.info(f"Fetching IO result from pod {pod_obj.name}")
```

Comment:
Maybe worth add this log in the get_fio_results method directly?
It will save all those new lines

Reply:
We already have that https://github.com/red-hat-storage/ocs-ci/pull/1194/files , I think the log can add some info about the stage it is in so that its more useful.

Reply:
If this log is just added in get_fio_results method, it will be duplicate message when  get_fio_rw_iops method is used.  So I feel it is better to move this log from 'get_fio_rw_iops' to 'get_fio_results' method. This will be more appropriate because it is 'get_fio_results' actually waiting for the result.

Reply:
Done